app/controllers/dashboards_controller.rb

```diff
@@ -5,6 +5,6 @@ def show
     @showing_all = params[:all_players].present?
     @game = current_user.build_game
     @leaderboard_users = User.by_elo
-    @leaderboard_users = @leaderboard_users.limit(10) unless @showing_all
+    @leaderboard_users = @leaderboard_users.rankable unless @showing_all
```

Comment:
how about a top 25 limit? otherwise if you're the 11th ranked player you either don't show up as ranked or, you are ranked something like 36th when viewing all players (with most of the users in the middle never having logged a game).